gempython/tools/optohybrid_user_functions_xhal.py

```diff
@@ -370,7 +370,7 @@ def performCalibrationScan(self, **kwargs):
         mask        - VFAT mask to use
         nevts       - Number of events for each dac value in scan
         outData     - Array of type c_uint32, if chan >= 0 array size
-                      must be: ((dacMax - dacMin + 1) / stepSize) * 24.
+                      must be: ((dacMax - dacMin + 1) / stepSize) * self.parentAMC.nVFAT.
```

Comment:
[same](https://github.com/cms-gem-daq-project/cmsgemos/pull/287/files#r297221217)...apply this comment everywhere.